src/main/java/in/projecteka/gateway/clients/model/ErrorCode.java

```diff
@@ -6,8 +6,8 @@
 import java.util.Arrays;
 
 public enum ErrorCode {
-    UNKNOWN_ERROR_OCCURRED(1000),
-    INVALID_TOKEN(1017),
+    UNKNOWN_ERROR_OCCURRED(2500),
+    INVALID_TOKEN(2401),
     SERVICE_DOWN(4222),
```

Comment:
This should be changed as well right? All gateway codes now start with 2xxx. 
Please change the code (and update the tracking sheet), and also check for mapping in other systems.